pnpm-lock.yaml

```diff
@@ -380,10 +380,11 @@ importers:
   sites/kit.svelte.dev:
     specifiers:
       '@sveltejs/adapter-auto': workspace:*
-      '@sveltejs/adapter-static': workspace:^1.0.0-next.28
+      '@sveltejs/adapter-static': workspace:^1.0.0-next.29
```

Comment:
The static adapter should presumably be completely removed from the kt.svelte.dev site.

Reply:
Maybe it's just my local, but after removing` @sveltejs/adapter-static`, `pnpm build` is failed.